### PR TITLE
Add linting script and lint files

### PR DESCRIPTION
**Pull request by martinratinaud**
Fixes https://github.com/ambanum/OpenTermsArchive/issues/833

### CODE REVIEW

File: declarations/YouTube.json [hunk 1]
```diff
@@ -12,63 +12,113 @@
...
+        ".ytr-link > svg"
+      ]
     },
     "Community Guidelines - Self-harm": {
       "fetch": "https://support.google.com/youtube/answer/2802245",
-      "select": [".article-container"],
-      "remove": [".scClientFeedbackInlineFeedbacklinkroot"]
+      "select": [
+        ".article-container"
+      ],
```
Comment (MattiSG) [inline declarations/YouTube.json:28]: I'm really not sure this format is an improvement 😕

**Review by MattiSG — CHANGES_REQUESTED** [review]
CI should fail on linting failure.

**Comment by martinratinaud** [comment]
Well I believe this can't be customized using `eslint-plugin-json-format`

For the sake of testing I tried

- testing on major onlines json linters

All lint the same way as this plugin does

- tried https://github.com/absolunet/eslint-config/tree/production/packages/json

```
npm install @absolunet/eslint-config-json
# update eslintrc config with extends "@absolunet/json"
# put an indent error in Aigle.json
./node_modules/eslint/bin/eslint.js ../france-declarations/declarations/Aigle.json -c ./.eslintrc.yaml  --fix
```
Format is not changed 

- tried https://github.com/azeemba/eslint-plugin-json
```
npm install eslint-plugin-json
# update eslintrc config with extends "plugin:json/recommended"
# put an indent error in Aigle.json
./node_modules/eslint/bin/eslint.js ../france-declarations/declarations/Aigle.json -c ./.eslintrc.yaml  --fix

```

The only one that seems to be doing the job seems to be prettier, will try to implement to see if it works

**Comment by MattiSG** [comment]
Thanks @martinratinaud for your research! As discussed, let's proceed with that setup and decide separately on how to improve the presentation of the linting. The main added value now will be unifying presentation, no matter how.

File: declarations/YouTube.json (hunk 1 above)
Comment (martinratinaud) [reply declarations/YouTube.json:28]: see below comment by @MattiSG

**Comment by Ndpnt** [comment]
Perhaps we should just avoid using an array for a single element, as the engine can handle both cases, with or without an array.

**Comment by MattiSG** [comment]
> Perhaps we should just avoid using an array for a single element

See https://github.com/ambanum/OpenTermsArchive/pull/841#issuecomment-1127320730

**Pull request merged**